### PR TITLE
add celery to project

### LINKED ISSUE #6: Celery bootstrap
Create a simple interface to working with Celery.

### PR DESCRIPTION
**Pull request by drgarcia1986**
fix #6

### CODE REVIEW

**Comment by drgarcia1986** [comment]
@renanivo I can write a simple test in `taskapp` app just to call a function with `delay` syntax, what do you think?

**Comment by renanivo** [comment]
@drgarcia1986 I think there's no need for that test. BTW, I never seen a celery setup done this way, it is very nice.

File: houserules/settings/development.py [hunk 1]
```diff
@@ -14,28 +14,31 @@
...
+)
 
-    # 3rd party apps
+THIRD_PARTY_APPS = (
     'allauth',
     'allauth.account',
     'allauth.socialaccount',
     'allauth.socialaccount.providers.github',
     'bootstrap3',
+    'houserules.taskapp.celery.CeleryConfig',
```
Comment (renanivo) [inline houserules/settings/development.py]: Is it a 3rd party or is it the project's celery app?

**Comment by renanivo** [comment]
I tried to run the celery tasks with:
```
celery worker -A houserules.taskapp -l debug
```

It failed with the error message:
```
django.db.utils.OperationalError: no such table: djkombu_queue
```

In order to use `django://` as the `BROKER_URL`, we need to add `kombu` to requirements and add `kombu.transport.django` to `INSTALLED_APPS`. Could you do that?

File: houserules/taskapp/celery.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import os
```
Comment (renanivo) [inline houserules/taskapp/celery.py]: What about renaming `tasksapp/celery.py` to `celery/apps.py`?

File: houserules/settings/development.py (hunk 1 above)
Comment (drgarcia1986) [inline houserules/settings/development.py]: I think it's a 3rd party app, what do you think?

**Comment by drgarcia1986** [comment]
@renanivo for sure, sorry about that

File: houserules/taskapp/celery.py (hunk 2 above)
Comment (drgarcia1986) [inline houserules/taskapp/celery.py]: Hum, let me think about usability in anothers apps:
```diff
- from houserules.tasksapp.celery import app
+ from houserules.celery.apps import app
```
It's not bad :smile:

**Comment by drgarcia1986** [comment]
@renanivo I'm trying to change module name, but it's happening some errors like this
```
AttributeError: module 'houserules.celery' has no attribute 'celery'
```

**Comment by renanivo** [comment]
Be sure to run `make clean` and try to start the workers with: `celery -A houserules.celery.apps worker -l info`

**Comment by drgarcia1986** [comment]
@renanivo It's worked, maybe I forgot of `make clean`

File: houserules/settings/development.py [hunk 3]
```diff
@@ -14,27 +14,31 @@
...
     'django.contrib.staticfiles',
+)
 
-    # 3rd party apps
+THIRD_PARTY_APPS = (
     'allauth',
     'allauth.account',
     'allauth.socialaccount',
     'allauth.socialaccount.providers.github',
+    'houserules.celery.apps.CeleryConfig',
```
Comment (renanivo) [inline houserules/settings/development.py]: It's your decision, but I still think it should be at project's apps. It is not _celery_ as an app, it is the houserules' setup of celery IMO.

**Comment by renanivo** [comment]
👍

File: houserules/settings/development.py (hunk 3 above)
Comment (drgarcia1986) [inline houserules/settings/development.py]: It's make a sense, but I still believe it's a simple Celery setup without any special behaviour.
But, it's one a line in settings file, I don't see any problem in change this in the future

**Pull request merged**